script/run_cluster.rb

```diff
@@ -47,7 +47,7 @@
 nodes = %w(aptrust chron hathi sdr tdr)
 
 def common_assignments(node)
-  "RAILS_ENV=impersonate IMPERSONATE=#{node} DATABASE_URL=sqlite3:db/impersonate_#{node}.sqlite3"
+  "RAILS_ENV=impersonate_#{node} IMPERSONATE=#{node} DATABASE_URL=sqlite3:db/impersonate_#{node}.sqlite3"
 end
```

Comment:
Note that the RAILS_ENV is now a node-specific environment so that it gets the correct content from `config/secrets` and `config/dpn`.